roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
+  - python-devel
+  - postgresql-devel
+
+- name: Add tmpfiles snippet
+  template:
+    dest: /etc/tmpfiles.d/softserve.conf
+    src: softserve.tmpfiles.conf
+  notify: Create systemd tmpfile
+
+- name: Add gunicorn systemd service
+  template:
+    dest: /etc/systemd/system/softserve.service
+    src: softserve.service
+  notify: Reload systemd
+
+- name: Start the softserve service
+  service:
+    name: softserve
+    enabled: True
+    state: started
+
+- name: Create the project directory
+  file:
+    path: '/home/softserve'
+    state: directory
+    mode: 0755
+
+- name: Create virtual env and install dependencies
+  pip:
+    virtualenv: '/home/softserve/env'
+    requirements: '/home/softserve/dev-requirements.txt'
+    virtualenv_python: python3.4
+
+- name: Configure Crontab
+  cron:
+    name: "shutdown check"
+    minute: "5"
+    job: "/home/softserve/env/bin/flask shutdown_check"
```

Comment:
So, for the unitiated like me, what does it do, and is running every hour at 5 minutes after the hour what is required ? (cause that seems oddly specific, and I wonder if you wanted "run it every 5 minutes" instead)